src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -712,7 +712,8 @@ public Token getFirstNonTightHtmlTag() {
          * @param messageKey message key
          * @param messageArguments message arguments
          */
-        ParseErrorMessage(int lineNumber, String messageKey, Object... messageArguments) {
+        /* default */ ParseErrorMessage(int lineNumber, String messageKey,
```

Comment:
@pbludov Not all `default`s are removed.

Reply:
Yes. Only top level classes are `public` now. Should this two be `public` also?